### PR TITLE
[DOCS-6122] Mobile workspace

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: mobile-workspace/latest/config/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Advanced Settings
+
+The advanced settings are preconfigured with the correct default settings to connect to Alfresco. Depending on your installation you may need to change some of these. The advanced settings can be accessed from the first screen. There are a number of options which help to authenticate the application with your Alfresco Content Services instance.
+
+| Field | Derscription |
+| ------- | --------- |
+| HTTPS | Allows you to switch between HTTP or HTTPS. If connecting to Alfresco Content Services Community Edition then HTTP is common but it is generally only used for testing purposes only. |
+| Port | The port used by your Alfresco Content Services instance. If you use HTTPS the default value is `443`, if it is HTTP the default value is `80`. |
+| Path | This is the domain name of your installation /alfresco, e.g. mydomain.com/alfresco. |
```
Comment (agologan) [inline mobile-workspace/latest/config/index.md:20]: The path to where Alfresco Content Services are located on your domain, defaulting to alfresco as in they are available at mydomain.com/alfresco

File: mobile-workspace/latest/config/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Configure Mobile Workspace
+---
+
+Find Alfresco Mobile Workspace at [Google Play](https://play.google.com/store/apps/details?id=com.alfresco.content.app&hl=en-GB&ah=tbrTPT50zRyrvFT0xxYx0IRH6DA){:target="_blank"} for Android or [Apple App Store](https://apps.apple.com/gb/app/alfresco-mobile-workspace/id1514434480){:target="_blank"} for iOS.
+
+## First screen
```
Comment (agologan) [inline mobile-workspace/latest/config/index.md:7]: Can we rename this to Login screen or Landing screen?

File: mobile-workspace/latest/config/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+| ------- | --------- |
+| HTTPS | Allows you to switch between HTTP or HTTPS. If connecting to Alfresco Content Services Community Edition then HTTP is common but it is generally only used for testing purposes only. |
+| Port | The port used by your Alfresco Content Services instance. If you use HTTPS the default value is `443`, if it is HTTP the default value is `80`. |
+| Path | This is the domain name of your installation /alfresco, e.g. mydomain.com/alfresco. |
+| Realm | When authenticating against Alfresco Content Services with the Identity Service configured the value in the realm field must match the `Realm` value in the Identity Service configuration. |
+| Client ID | This value must match the value the `Client ID` value configured in the Identity Service. |
+
+### SSO (Identity Service)
+
+If you are connecting to Alfresco Content Services using the Identity Service the app will request the credentials for your identity provider, e.g. okta, including any multi-factor authentication steps.
```
Comment (agologan) [inline mobile-workspace/latest/config/index.md:26]: [...] the app will delegate authentication responsibilities to your identity provider, e.g. okta [...]

File: mobile-workspace/latest/using/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: Using Mobile Workspace
+---
+
+## Searching
+
+* Search is persistent and the Mobile Workspace allows global searching at any time. The results can be quickly filtered by files, folders or libraries just like in the Digital Workspace.
```
Comment (agologan) [inline mobile-workspace/latest/using/index.md:7]: Although I understand what you mean I wouldn't call search as being persistent in the docs I'd rather just say:
Global search is available from all of the main/top level screens allowing search at any time.

File: mobile-workspace/latest/using/index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* The Mobile Workspace keeps a recent history of search terms so that previous searches can be quickly repeated.
+* Contextual search is available within folders. This search mode will only search within that folder but can be converted to a global search using the quick filter. You can also filter by files and folders within the contextual folder search.
+
+## Recent Files
+
+The Recent Files view shows all the files that have been created or modified within the last 30 days by the logged in user. The Search API is used to query SOLR for changes made by the user and includes an extra column to display where the file is located in the content repository.
```
Comment (agologan) [inline mobile-workspace/latest/using/index.md:13]: Please drop SOLR as there may alternative search providers used. Maybe rewrite it as:
The Search API is used to query for changes made by the user [...]

File: mobile-workspace/latest/using/index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+Offline is a collection of content that is individual to a user and is available when offline.
+The Mobile Workspace will attempt to keep this collection updated with the latest version wherever possible.
+
+## Browse
+
+Browse contains the rest of the navigation structure. You can access your libraries, personal files, shared files and discarded files from this screen.
+
+* Personal Files retrieves all content from the logged in user's home area `/User Homes/<username>/` in the repository. If the user is an administrator who does not have a home folder then the repository root folder is shown.
+
+* File Libraries retrieves all the sites that the user is a member of including what type of site it is, either public, moderated or private. File Libraries is the Libraries component, using the Sites API.
```
Comment (agologan) [inline mobile-workspace/latest/using/index.md:30]: We don't show library type type, can we remove that part for now?

File: mobile-workspace/latest/config/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/config/index.md:20]: Done

File: mobile-workspace/latest/config/index.md (hunk 2 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/config/index.md:7]: done

File: mobile-workspace/latest/config/index.md (hunk 3 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/config/index.md:26]: Done

File: mobile-workspace/latest/using/index.md (hunk 4 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/using/index.md:7]: Done

File: mobile-workspace/latest/using/index.md (hunk 5 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/using/index.md:13]: done

File: mobile-workspace/latest/using/index.md (hunk 6 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/using/index.md:30]: done

**Review by agologan — DISMISSED** [review]

**Review by m-brouillette — DISMISSED** [review]

File: mobile-workspace/latest/index.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+title: Alfresco Mobile Workspace
+---
+
+Alfresco Mobile Workspace enables you to have a seamless experience across multiple platforms because it gives you the ability to work away from your workstation without compromising the way you access content. You can view your libraries, recent and shared files, and manage your favorites from your mobile device. You can view all major document types including Microsoft Word, Excel, and PowerPoint. You can also view images, including but not limited to `JPEG` and `PNG`. You can use the Mobile Workspace to manage and view content while offline and it has a dark mode to support your well being.
```
Comment (anxumalo) [inline mobile-workspace/latest/index.md:5]: well-being?

File: mobile-workspace/latest/config/index.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+## Login screen
+
+From the first screen you enter the URL of your Alfresco Content Services instance into the `Connect to`  field. You will then be asked to authenticate using your Alfresco credentials. If you want to use Single Sign-On (SSO) to connect to the Mobile Workspace you can enter the URL of your Alfresco Identity Management Service. You can enter the URL in the following format `my.alfresco.com`.
+> **Note:** You do not need to enter www because the URL will be formatted by the application.
+
+### Advanced Settings
+
+The advanced settings are preconfigured with the correct default settings to connect to Alfresco. Depending on your installation you may need to change some of these. The advanced settings can be accessed from the first screen. There are a number of options which help to authenticate the application with your Alfresco Content Services instance.
+
+| Field | Derscription |
```
Comment (anxumalo) [inline mobile-workspace/latest/config/index.md:16]: Description

File: mobile-workspace/latest/config/index.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+If you are connecting to Alfresco Content Services using the Identity Service the app will delegate authentication responsibilities to your identity provider, for example okta, including any multi-factor authentication steps.
+
+For Single Sign On (SSO) information see the [Single Sign On (SSO) Guide]({% link identity-service/latest/tutorial/sso/index.md %}){:target="_blank"}.
+
+## Extending
+
+You can extend the Mobile Workspace.
+
+Find the Mobile Workspace code at the following github repositories for [Android](https://github.com/alfresco/alfresco-mobile-workspace-android){:target="_blank"} or [iOS](https://github.com/alfresco/alfresco-mobile-workspace-ios){:target="_blank"}.
```
Comment (anxumalo) [inline mobile-workspace/latest/config/index.md:34]:
```suggestion
Find the Mobile Workspace code in the following GitHub repositories:

* [Android](https://github.com/alfresco/alfresco-mobile-workspace-android){:target="_blank"}
* [iOS](https://github.com/alfresco/alfresco-mobile-workspace-ios){:target="_blank"}

```

File: mobile-workspace/latest/using/index.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+## Searching
+
+* You can search globally from all of the main/top level screens which means you can search at any time.
+* The Mobile Workspace keeps a recent history of search terms so that previous searches can be quickly repeated.
+* Contextual search is available within folders. This search mode will only search within that folder but can be converted to a global search using the quick filter. You can also filter by files and folders within the contextual folder search.
+
+## Recent Files
+
+The Recent Files view shows all the files that have been created or modified within the last 30 days by the logged in user. The Search API is used to query for changes made by you and includes an extra column to display where the file is located in the content repository.
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:13]: I'd suggest using `**Recent Files**` for this and any other UI term in the rest of this page.

File: mobile-workspace/latest/config/index.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+---
+title: Configure Mobile Workspace
+---
+
+Find Alfresco Mobile Workspace at [Google Play](https://play.google.com/store/apps/details?id=com.alfresco.content.app&hl=en-GB&ah=tbrTPT50zRyrvFT0xxYx0IRH6DA){:target="_blank"} for Android or [Apple App Store](https://apps.apple.com/gb/app/alfresco-mobile-workspace/id1514434480){:target="_blank"} for iOS.
+
+## Login screen
+
+From the first screen you enter the URL of your Alfresco Content Services instance into the `Connect to`  field. You will then be asked to authenticate using your Alfresco credentials. If you want to use Single Sign-On (SSO) to connect to the Mobile Workspace you can enter the URL of your Alfresco Identity Management Service. You can enter the URL in the following format `my.alfresco.com`.
```
Comment (anxumalo) [inline mobile-workspace/latest/config/index.md:9]: `Alfresco Identity Management Service` isn't the product name that's used in the Docs site and may confuse customers. Should this be just 'Identity Service` so it matches the later reference (line 22)?

File: mobile-workspace/latest/using/index.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+Browse contains the rest of the navigation structure. You can access your libraries, personal files, shared files and discarded files from this screen.
+
+* Personal Files retrieves all content from the logged in user's home area `/User Homes/<username>/` in the repository. If the user is an administrator who does not have a home folder then the repository root folder is shown.
+
+* File Libraries retrieves all the sites that the user is a member of, either public, moderated or private. File Libraries is the Libraries component, using the Sites API.
+
+* The Shared Files view aggregates all files that have been shared using the QuickShare feature in the content repository.
+
+* The Trash view shows all the items that a user has deleted, an administrator can see items deleted by all users. The actions available in this view are Restore and Permanently Delete.
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:34]:
```suggestion
* The Trash view shows all the items that a user has deleted. An administrator can see items deleted by all users. The actions available in this view are **Restore** and **Permanently Delete**.
```

**Review by anxumalo — COMMENTED** [review]
I can see that this has been reviewed already but I have some suggested updates. We can discuss the logo issue offline :-)

File: mobile-workspace/latest/config/index.md (hunk 8 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/config/index.md:16]: Corrected !

File: mobile-workspace/latest/using/index.md (hunk 10 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/using/index.md:13]: ok, done.

File: mobile-workspace/latest/config/index.md (hunk 11 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/config/index.md:9]: done

File: mobile-workspace/latest/index.md (hunk 7 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/index.md:5]: done

File: _data/home.yaml [hunk 13]
```diff
@@ -137,6 +137,10 @@ applications:
     text: "Alfresco Content Services for mobile allows you to view, edit, and share information in and out of the office via the mobile app."
     url: /content-mobile/latest/
     image: /assets/img/logo_cm_72px.svg
+  - title: "Mobile Workspace"
+    text: "Alfresco Mobile Workspace enables users to work away from their workstation without compromising the way they access content. Keep productivity high by transporting technical documents into the field without having to worry about a data connection."
+    url: /mobile-workspace/latest/
+    image: /assets/img/logo_cm_72px.svg
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply _data/home.yaml]: resolved by adjusting the home page

File: _config.yml [hunk 14]
```diff
@@ -791,6 +791,19 @@ defaults:
...
+    values:
+      layout: "docs"
+      product: "Alfresco Mobile Workspace"
+      location: "mobile-workspace"
+      toc: "mobile-workspace"
+  - scope:
+      path: "mobile-workspace/latest"
+    values:
+      version: 1.0
+      latest: true
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply _config.yml]: adjusted

File: _data/home.yaml (hunk 13 above)
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply _data/home.yaml]: done

File: mobile-workspace/latest/using/index.md (hunk 10 above)
Comment (anxumalo) [reply mobile-workspace/latest/using/index.md]: I'm writte nthe  **text descriptions** for UI items in bold, and reverted the changes to the headings.

**Review by anxumalo — APPROVED** [review]

**Pull request merged**